web/src/components/feature/PhaseTable.tsx

```diff
@@ -19,18 +30,36 @@ function getRows(
 
 export const PhaseTable = (props: {
   multiFlashResponse: MultiflashResponse
-  cubicFeedFlow: number
 }) => {
-  const { multiFlashResponse, cubicFeedFlow } = props
+  const { multiFlashResponse } = props
+  const [feedFlow, setFeedFlow] = useState(1000)
+  const [feedUnit, setFeedUnit] = useState('Sm3/d')
+
+  const onChangeFeedUnit = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setFeedUnit(event.target.checked ? 'kg/d' : 'Sm3/d')
+  }
+
+  const onChangeFeedFlow = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setFeedFlow(Number(event.target.value))
+  }
+
   return (
-    <DynamicTable
-      headers={[
-        'Phases',
-        'Mass Concentration',
-        'Mole Concentration',
-        'Mercury Flow (g/d)',
-      ]}
-      rows={getRows(multiFlashResponse, cubicFeedFlow)}
-    />
+    <PhaseTableContainer>
+      <FeedFlowInput
+        feedFlow={feedFlow}
+        feedUnit={feedUnit}
+        onChangeFeedFlow={onChangeFeedFlow}
```

Comment:
This is a matter if taste. But IMO, a simple "setState" should be done inline. Results in less jumping around when reading the code.
`onChangeFeedFlow={(...)=>setState(Number(event.target.value))}`